ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -363,10 +368,20 @@ private void initializeInstances(List<InstanceConfig> instanceConfigs) throws Ex
      */
     private void updateSealedStateOfReplicas(List<InstanceConfig> instanceConfigs) {
       for (InstanceConfig instanceConfig : instanceConfigs) {
-        AmbryDataNode node = instanceNameToAmbryDataNode.get(instanceConfig.getInstanceName());
-        HashSet<String> sealedReplicas = new HashSet<>(getSealedReplicas(instanceConfig));
-        for (AmbryReplica replica : ambryDataNodeToAmbryReplicas.get(node)) {
-          replica.setSealedState(sealedReplicas.contains(replica.getPartitionId().toPathString()));
+        int schemaVersion = getSchemaVersion(instanceConfig);
+        if (schemaVersion == 0) {
+          AmbryDataNode node = instanceNameToAmbryDataNode.get(instanceConfig.getInstanceName());
+          if (node == null) {
+            logger.info("Dynamic addition of new nodes is not yet supported, ignoring InstanceConfig {}",
```

Comment:
info v/s warn?

Reply:
info seemed apt as this is not severe enough for a warning. Likelihood of new node additions is low enough that this did not have to be a trace.